packages/react/src/Trans.test.tsx

```diff
@@ -54,17 +54,29 @@ describe("Trans component", function () {
     console.error = originalConsole
   })
 
-  it("should throw a console.error if using twice props", function () {
-    const originalConsole = console.error
-    console.error = jest.fn()
+  it("should log a console.error if using `render` and `component` props at the same time", () => {
+    const RenderChildrenInSpan = ({ children }: TransRenderProps) => (
+      <span>{children}</span>
+    )
 
-    // @ts-expect-error testing the error
-    renderWithI18n(<Trans render="div" component="span" id="Some text" />)
-    expect(console.error).toHaveBeenCalled()
-    console.error = originalConsole
+    const consoleErrorSpy = jest.spyOn(console, "error")
```

Comment:
will this print error in the console while testing? If yes - it's not good. These errors, very annoying when you run full suite